ACM_General/events/models.py

```diff
@@ -47,6 +47,46 @@ def get_path_for_flier(instance, filename):
     )
 
 
+class Tag(models.Model):
+    """
+    Class used for adding classification tags on events.
+    """
+
+    #: The name of the tag that is displayed on the event creation form; represented
+    #: as a CharField.
+    display_name = models.CharField(
+        verbose_name=_('Event Tag'),
+        help_text=_('Different categories for the event.'),
```

Comment:
This help text is not really descriptive as to what it does. Should be something along the lines of `The name of the Tag that will be displayed to users`